events/awards/README.md

```diff
@@ -15,4 +17,4 @@ These awards are handed out during the end-of-year Contributor Summit.
 [playbook]: ./README.md
 [2021-awards-recipients]: ./award-recipients/2021-award-recipients.md
 [2020-awards-recipients]: ./award-recipients/2020-award-recipients.md
-[2019-awards-recipients]: ./award-recipients/2019-award-recipients.md
\ No newline at end of file
+[2019-awards-recipients]: ./award-recipients/2019-award-recipients.md
```

Comment:
If we're fixing this, why have these links (and the related files) at all?  Why not just direct to the awards page?

Reply:
Updated. Like this?

Reply:
ACK